bc_obps/reporting/api/report_review_changes.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpRequest
+from service.error_service.custom_codes_4xx import custom_codes_4xx
+from reporting.constants import EMISSIONS_REPORT_TAGS
+from reporting.schema.generic import Message
+from .permissions import approved_industry_user_report_version_composite_auth
+from ..models import ReportVersion
+from ..schema.report_final_review import ReportVersionSchema
+from .router import router
+from service.report_version_service import ReportVersionService
+from reporting.service.report_review_changes_service import ReportReviewChangesService
+
+
+@router.get(
+    "/report-version/{version_id}/diff-data",
+    response={200: dict, custom_codes_4xx: Message},
+    tags=EMISSIONS_REPORT_TAGS,
+    description="Fetch serialized data for the given report version and the latest previous version with the same report_id.",
```

Comment:
I don't think this description quite matches the actual behaviour of the function - e.g., the difference in behaviour if a value is provided for `compare_version_id` or not